keras_cv/layers/preprocessing/base_image_augmentation_layer_test.py

```diff
@@ -146,7 +146,7 @@ def test_augment_leaves_batched_extra_dict_entries_unmodified(self):
     def test_augment_image_and_localization_data(self):
```

Comment:
lets write a unit test to make sure it raises an error when the class axis is missing.

Reply:
done!